### PR TITLE
Add relative labels

### PR DESCRIPTION
**Pull request by Snektron**
Adds relative labels to the assembler, as well as
fix some windows related stuff

This time without exploding my branch

### CODE REVIEW

File: assembler/assembler.c [hunk 1]
```diff
@@ -27,6 +25,15 @@
...
 struct assembler_state state;
 
+int log10_u64(uint64_t i) {
+	int log = 1;
+	while (i >= 10) {
+		i /= 10;
+	}
+
+	return log;
+}
```
Comment (SirCmpwn) [inline assembler/assembler.c]: This function always returns 1.

**Comment by SirCmpwn** [comment]
Looks good aside from the one issue.

**Comment by SirCmpwn** [comment]
Thanks!

**Comment by Snektron** [comment]
Must have slipped me, fixed now

**Pull request merged**